Add guideline B4c+ (BLD sight blocker timing)

hanwu85 opened the pull request:
The judge should put up the sight blocker after the competitors have applied the first move, because competitors can remove the blindfold if they have not applied any moves.

Thanks @denthebro for pointing out this. #948

hanwu85 committed:
add guideline B4c+ (BLD sight blocker timing)

The judge should put up the sight blocker after the competitors have applied the first move, because competitors can remove the blindfold if they have not applied any moves.

Ivan-Brigidano reviewed (approved):
It sounds reasonable to me. But I would like to see more opinions from other WRC members before merging it into the Draft

mirandapedro commented:
Hi.

Taking a closer look at the Regulations, I've found some inconsistencies regarding when should the judge put up the the blocker between the competitor and the puzzle.

We're normally taught that it should only be done after the competitor applies the first move to the puzzle. This is in order to follow B4e, where a competitor may undon the blindfold before they've made the first move.

Following this line of thought, the guideline here being proposed is fine, and IMO should only be and "CLARIFICATION", if we understand it is only following what is in the Regulations.

However, B4c2 states the jughe should place the objetc while the competitor is wearing the blindfold. If we follow this, after the competitor has donned the blindfold, but before he has made his first move, the judge should still be using the blocker.

This is unnecessary, as the competitor can go back to the memorization phase and see the puzzle any time he wants, and clearly doesn't reflect what happens in competitions.

My suggestion is to remove B4c2 and add the proposed guideline as an article in the Regulations, perhaps as an edit to B4c's current wording. It could look like this:

"B4c) After the competitor has applied the first move to the puzzle, the judge must ensure that there is a sight blocker between the competitor's face and the puzzle.

mirandapedro reviewed (changes requested):
Hi.

Taking a closer look at the Regulations, I've found some inconsistencies regarding when should the judge put up the the blocker between the competitor and the puzzle.

We're normally taught that it should only be done after the competitor applies the first move to the puzzle. This is in order to follow B4e, where a competitor may undon the blindfold before they've made the first move.

Following this line of thought, the guideline here being proposed is fine, and IMO should only be and "CLARIFICATION", if we understand it is only following what is in the Regulations.

However, B4c2 states the jughe should place the objetc while the competitor is wearing the blindfold. If we follow this, after the competitor has donned the blindfold, but before he has made his first move, the judge should still be using the blocker.

This is unnecessary, as the competitor can go back to the memorization phase and see the puzzle any time he wants, and clearly doesn't reflect what happens in competitions.

My suggestion is to remove B4c2 and add the proposed guideline as an article in the Regulations, perhaps as an edit to B4c's current wording. It could look like this:

"B4c) After the competitor has applied the first move to the puzzle, the judge must ensure that there is a sight blocker between the competitor's face and the puzzle.

Commit:
Clearing up the use of sight block in Blindfolded Solving

We're normally taught that the should only be used after the competitor applies the first move to the puzzle. This is in order to follow B4e, where a competitor may undon the blindfold before they've made the first move.

However, B4c2 states the jugde should place the objetc while the competitor is wearing the blindfold. If we follow this, after the competitor has donned the blindfold, but before he has made his first move, the judge should still be using the blocker.

This is unnecessary, as the competitor can go back to the memorization phase and see the puzzle any time he wants, and clearly doesn't reflect what happens in competitions.

Besides, B4c1 states that the competitor must wear the blindfold such that their view of the puzzle would still clearly be blocked if the sight blocker were not in the way. Therefore using a sight blocker before the competitor has applied the first move to the puzzle is unnecessary.

This commit removes B4c2 and the proposed guideline, and instead adapts the wording of the Regulation B4c.

In this way, we make it clearer in the Regulations when the sight blocker should be used, without adding a Guideline when it is not necessary.

mirandapedro commented:
If @Ivan-Brigidano and @hanwu85 agree to these changes I believe we can merge it into the draft.

hanwu85 commented:
I think the purpose of B4c2 is to prevent a DNF when a judge forgets to put the sight blocker for a short period of time. (eg, 1 second)

mirandapedro commented:
But how does it do that? 

Regulation B4c2 was added on [commit b2802c9](https://github.com/thewca/wca-regulations/commit/b2802c923fc2b88a8d72fc7b0d30940fd691204a).

I believe it's original purpose was to have the judge place the sight blocker between the competitor and the puzzle at all times, even before the competitor has made applied the first move to the puzzle. 

Even then, when the commit was made, B4c and B4c2 were contradictory. I don't think there's a need for the Regulation B4c2.

lgarron commented on wca-regulations.md:
```diff
@@ -446,9 +446,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - B4a) The competitor dons the blindfold to start the blindfolded phase.
     - B4b) The competitor must not apply moves to the puzzle before they have fully donned the blindfold. Penalty: disqualification of the attempt (DNF).
-    - B4c) The judge must ensure that there is a sight blocker between the competitor's face and the puzzle while the competitor is solving.
+    - B4c) After the competitor has applied the first move to the puzzle, the judge must ensure that there is a sight blocker (e.g. a sheet of paper or cardboard) between the competitor's face and the puzzle.
```
A small nit: "after" technically applies to the entire solving period. We can use an alternative that makes it clear that the goal is to cover during as much of the solving period as possible.

```suggestion
    - B4c) As soon as the competitor applies the first move to the puzzle, the judge must ensure that there is a sight blocker (e.g. a sheet of paper or cardboard) between the competitor's face and the puzzle.
```

lgarron reviewed (commented):
One small nit, but I'd love to see this landed!

Ivan-Brigidano committed:
Update wca-regulations.md

Co-authored-by: Lucas Garron <lgarron@worldcubeassociation.org>
